packages/core/data-transfer/lib/engine/index.ts

```diff
@@ -133,34 +133,22 @@ class TransferEngine implements ITransferEngine {
   }
 
   async transferEntities(): Promise<void> {
-    // const inStream = await this.sourceProvider.streamEntities?.();
-    // const outStream = await this.destinationProvider.getEntitiesStream?.();
-    // if (!inStream || !outStream) {
-    //   console.log('Unable to transfer entities, one of the stream is missing');
-    //   return;
-    // }
-    // return new Promise((resolve, reject) => {
-    //   pipeline(
-    //     inStream,
-    //     chain([
-    //       (data) => {
-    //         console.log('hello', data);
-    //         return data;
-    //       },
-    //     ]),
-    //     outStream,
-    //     (e: NodeJS.ErrnoException | null) => {
-    //       if (e) {
-    //         console.log('Something wrong happened', e);
-    //         return reject(e);
-    //       }
-    //       console.log('All the entities have been transferred');
-    //       resolve();
-    //     }
-    //   );
-    // });
-    console.log('transferEntities not yet implemented');
-    return new Promise((resolve) => resolve());
+    const inStream = await this.sourceProvider.streamEntities?.();
```

Comment:
I'm just thinking: should source be something like `getReadableEntitiesStream` and destination `getWritableEntitiesStream`?

Reply:
Indeed that could make sense. Methods naming is still up to debate, I've called them like that initially to have a sense of "I stream the entities to" and "I receive the entities from."

I would suggest we debate from the specifications & update them before modifying the actual implementation. What do you think?